app/master/build_preparer.py

```diff
@@ -9,7 +9,7 @@
 from app.project_type.project_type import ProjectType
 
 
-class SerialRequestHandler(object):
+class BuildPreparer(object):
```

Comment:
I'm not married to this name. I generally don't like verby nouns as class names.